src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -76,54 +96,115 @@ export const MyFieldAndBackground = () => {
         sx={{
           bgcolor: 'info.light',
         }}>
-        <Typography variant="h2">{t('my_page.my_profile.background')}</Typography>
         <Formik initialValues={initialValues} onSubmit={(values) => updatePerson.mutate(values)} enableReinitialize>
-          {({ isSubmitting, dirty }: FormikProps<PersonBackgroundFormData>) => (
-            <Form>
-              <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', mt: '1rem' }}>
-                <Typography variant="h3">{t('my_page.my_profile.field_and_background.norwegian')}</Typography>
-                <Field name={'background.no'}>
-                  {({ field }: FieldProps<string>) => (
-                    <>
-                      <TextField
+          {({ isSubmitting, dirty, setFieldValue, resetForm }: FormikProps<PersonBackgroundFormData>) => (
+            <>
+              <Form>
+                <Box>
+                  <Typography variant="h2">{t('my_page.my_profile.field_and_background.field')}</Typography>
+                  <Typography variant="h3" sx={{ mb: '1rem', mt: '1.5rem' }}>
+                    {t('my_page.my_profile.field_and_background.field_text')}
+                  </Typography>
+                  <Field name={'keywords'}>
+                    {({ field }: FieldProps<Keywords[]>) => (
+                      <Autocomplete
                         {...field}
-                        inputProps={{ maxLength: 200 }}
-                        label={t('my_page.my_profile.background')}
-                        variant="filled"
-                        multiline
-                        rows="3"
-                        placeholder={t('my_page.my_profile.field_and_background.background_placeholder')}
+                        loading={fieldQuery.isFetching}
+                        value={field.value ?? []}
+                        multiple
+                        options={keywordsResult}
+                        isOptionEqualToValue={(option, value) =>
+                          getLanguageString(option.label) === getLanguageString(value.label)
```

Comment:
bedre å sjekke likhet med IDer enn tekstrepresentasjoner som endrer seg basert på språk
```suggestion
                          option.type === value.type
```